doc/Language/list.pod6

```diff
@@ -480,14 +480,17 @@ which is selecting the 4 to 6th element from the three  first dimensions (C<^3>)
 A L<C<Range>|/type/Range> is a container for a lower and an upper boundary.
 Generating a slice with a C<Range> will include any index between those bounds,
 including the bounds. For infinite upper boundaries we agree with
-mathematicians that C<Inf> equals C<Inf-1>.
+mathematicians that C<Inf> equals C<Inf-1>. Slicing with an infinite Range
+will L<truncate|/language/subscripts#Truncating_slices>.
```

Comment:
Not clear the relationship between the two sentences here. If it's "so,..." they the example contradicts them. it's only the combination of -1 and using the ^ which truncates the range. How does that follow from the two sentences?

Reply:
There is no causal relationship between the two. I just found the information about truncation was missing from this discussion.

To fix this properly, without just removing it again, maybe this entire section about Range slicing should be taken off the List context section there and instead go into http://docs.perl6.org/language/subscripts#Slices. What do you think?

Reply:
Maybe you could move it up in the paragraph, in the sentence starting with "Generating a slice"